frontend/src/component/admin/network/NetworkTrafficUsage/NetworkTrafficUsage.tsx

```diff
@@ -245,10 +247,7 @@ const NewNetworkTrafficUsage: FC = () => {
     );
 
     const estimatedMonthlyCost = calculateEstimatedMonthlyCost(
-        chartDataSelection.grouping === 'daily'
-            ? chartDataSelection.month
-            : currentMonth,
-        data.datasets,
+        traffic.usage?.apiData,
```

Comment:
This ? is because `usage` might be undefined. This is because the hook doesn't account for whether you have data or not. I'll address this in (yet another) followup.